tutorials/nlp/Relation_Extraction-BioMegatron.ipynb

```diff
@@ -0,0 +1,601 @@
+    "\n",
+    "# the exp_dir provides a path to the current experiment for easy access\n",
+    "exp_dir = str(exp_dir)\n",
+    "exp_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8tjLhUvL_o7_"
+   },
+   "source": [
+    "Before initializing the model, we might want to modify some of the model configs. For example, we might want to modify the pretrained BERT model and use [Megatron-LM BERT](https://arxiv.org/abs/1909.08053) or [AlBERT model](https://arxiv.org/abs/1909.11942):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "Xeuc2i7Y_nP5"
+   },
+   "outputs": [],
+   "source": [
+    "# complete list of supported BERT-like models\n",
+    "print(nemo_nlp.modules.get_pretrained_lm_models_list())\n",
+    "\n",
+    "# specify BERT-like model, you want to use, for example, \"megatron-bert-345m-uncased\" or 'bert-base-uncased'\n",
+    "PRETRAINED_BERT_MODEL = \"biomegatron-bert-345m-uncased\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "RK2xglXyAUOO"
+   },
+   "outputs": [],
+   "source": [
+    "# add the specified above model parameters to the config\n",
+    "config.model.language_model.pretrained_model_name = PRETRAINED_BERT_MODEL"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "fzNZNAVRjDD-"
+   },
+   "source": [
+    "Now, we are ready to initialize our model. During the model initialization call, the dataset and data loaders we'll be prepared for training and evaluation.\n",
+    "Also, the pretrained BERT model will be downloaded, note it can take up to a few minutes depending on the size of the chosen BERT model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "NgsGLydWo-6-"
+   },
+   "outputs": [],
+   "source": [
+    "model = nemo_nlp.models.TextClassificationModel(cfg=config.model, trainer=trainer)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "kQ592Tx4pzyB"
+   },
+   "source": [
+    "## Monitoring training progress\n",
+    "Optionally, you can create a Tensorboard visualization to monitor training progress.\n",
+    "If you're not using Colab, refer to [https://www.tensorflow.org/tensorboard/tensorboard_in_notebooks](https://www.tensorflow.org/tensorboard/tensorboard_in_notebooks) if you're facing issues with running the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "mTJr16_pp0aS"
+   },
+   "outputs": [],
+   "source": [
+    "# load the TensorBoard notebook extension\n",
+    "%load_ext tensorboard\n",
+    "%tensorboard --logdir {exp_dir}"
+   ]
```

Comment:
Please see the update for this cell in other notebooks - there might be issues with the tensorboard when the notebook is used outside of colab